Examples/MAX78000/FreeRTOSDemo/project.mk

```diff
@@ -16,6 +16,11 @@ DEBUG=1
 
 LIB_FREERTOS = 1
 
+# Can provide a value for the FREERTOS heap allocation scheme
```

Comment:
Instead of documenting this here, you can add some info to the UG.

See [here](https://github.com/analogdevicesinc/msdk/blob/main/USERGUIDE.md#sdhc-build-variables) for an example for the SDHC library.

The FreeRTOS lib hasn't had any options before, so I'd suggest adding a similar table

Reply:
Good idea! I'll update once this has been added.

Reply:
Should be good now 👍